gbus/abstractions.go

```diff
@@ -205,6 +205,7 @@ type TxProvider interface {
 	New() (*sql.Tx, error)
 	Dispose()
 	Ping(timeoutInSeconds time.Duration) bool
+	GetDb() *sql.DB
```

Comment:
I think we can remove the GetDb function and pass the migration function an instance of the db struct coming from Provider.Database